src/Calculator/Views/MainPage.xaml.cpp

```diff
@@ -115,6 +115,14 @@ void MainPage::WindowSizeChanged(_In_ Platform::Object ^ /*sender*/, _In_ Window
 {
     // We don't use layout aware page's view states, we have our own
     UpdateViewState();
+    if (m_model->CalculatorViewModel->IsAlwaysOnTop)
+    {
+        Windows::Storage::ApplicationDataContainer ^ localSettings = Windows::Storage::ApplicationData::Current->LocalSettings;
+        // double width = safe_cast<float>(this->ActualWidth);
+        // double height = safe_cast<float>(this->ActualHeight);
+        localSettings->Values->Insert(App::WidthLocalSettings, this->ActualWidth);
+        localSettings->Values->Insert(App::HeightLocalSettings, this->ActualHeight);
```

Comment:
Suspend may not always be called when the app exits. For example, it may be forcefully exited by the system.

Reply:
Most apps save their state during suspend. You can also read docs here:
https://docs.microsoft.com/en-us/windows/uwp/launch-resume/suspend-an-app#save-application-data-before-suspension

Unless there is something special about this setting, I think suspend should cover most cases we need. If the system forcefully causes the app to shutdown then I think I would rather  lose the setting data rather than degrading performance on resize. @mcooley thoughts on this?

Reply:
Most apps save their state during suspend. You can also read docs here:
https://docs.microsoft.com/en-us/windows/uwp/launch-resume/suspend-an-app#save-application-data-before-suspension

Unless there is something special about this setting, I think suspend should cover most cases we need. If the system forcefully causes the app to shutdown then I think I would rather  lose the setting data rather than degrading performance on resize. @mcooley thoughts on this?